src/test/pages/TeacherRegistration.test.tsx

```diff
@@ -19,6 +19,11 @@ import { registrationMockUser, registrationUserData } from "../../mocks/data";
 const registerUserSpy = jest.spyOn(actions, "registerUser");
 const upgradeAccountSpy = jest.spyOn(actions, "upgradeAccount");
 
+const clickThroughOptions = async () => {
```

Comment:
This name should reflect the fact that you're following the Teacher path, just to make this more descriptive.

Reply:
Renamed this one to `confirmTeacherAndAcceptTerms`